ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/snapsync/SnapWorldStateDownloader.java

```diff
@@ -124,13 +124,13 @@ public CompletableFuture<Void> run(
       final BlockHeader header = fastSyncState.getPivotBlockHeader().get();
       final Hash stateRoot = header.getStateRoot();
       LOG.info(
-          "Downloading world state from peers for block {}. State root {} pending request {}",
+          "Downloading world state from peers for pivot block {}. State root {} pending request {}",
```

Comment:
should that not be "pending request**s**"?

Reply:
fixed